techniques/systemSettings/systemManagement/cronManagement/3.0/metadata.xml

```diff
@@ -14,6 +14,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <TECHNIQUE name="Cron daemon configuration">
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```

Comment:
Hmmm, I don't think we can reasonably *deprecate* versions of techniques in the same major version as they were supported in! This should be added in master branch instead.